app/controllers/course/course_settings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Course::CourseSettingsController < Course::ModuleController
+  layout 'course_settings'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.